.codegen/impl.go.tmpl

```diff
@@ -18,6 +18,10 @@ type {{.CamelName}}Impl struct {
 	client *client.DatabricksClient
 }
 
+func (a *{{.CamelName}}Impl) Client() client.DatabricksClientInterface {
+	return a.client
```

Comment:
we don't need to do this anymore - there's HTTPTransport mocking - see so DatabricksClient doesn't really have to be mocked - we can mock out the HTTP transport - https://github.com/databrickslabs/watchdog/blob/main/scan/scan_test.go#L29-L72

and if you need direct files - see https://github.com/databrickslabs/watchdog/blob/main/scan/notebooks/content_scanner_test.go#L46-L48